misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -17,6 +17,9 @@ data class WebConfig(
 
   val ssl: WebSslConfig? = null,
 
+  /** Configuration to enable Jetty to listen for traffic being relayed through a sidecar (like Envoy). */
+  val sidecar: WebSidecarConfig? = null,
```

Comment:
Since sidecar is an implementation choice, consider making naming a little more generic:
`val unix_domain_socket: UnixDomainSocketConfig? = null`

```
skim:
  web:
    port: 8080
    ssl:
      port: 8443
    unix_domain_socket:
      path: /foo/bar
```
```